receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -648,6 +648,45 @@ func (s *MongoDBAtlasClient) GetAlerts(ctx context.Context, groupID string, opts
 	return alerts.Results, hasNext(response.Links), nil
 }
 
+// GetEventsOptions are the options to use for making a request to get Project Events
+type GetEventsOptions struct {
+	// Which page of the paginated events
+	PageNum int
+	// How large the Pages will be
+	PageSize int
+	// The list of Event Types https://www.mongodb.com/docs/atlas/reference/api/events-projects-get-all/#event-type-values
+	// to grab from the API
+	EventTypes []string
+	// The oldest date to look back for the events
+	MinDate time.Time
+	// the newest time to accept events
+	MaxDate time.Time
+}
+
+// GetEvents returns the events specified for the set projects
+func (s *MongoDBAtlasClient) GetEvents(ctx context.Context, groupID string, opts *GetEventsOptions) (ret []*mongodbatlas.Event, nextPage bool, err error) {
```

Comment:
In the calling function, this is project ID, and group ID appears to mean something else.

Reply:
Was just following the convention of the previous function signatures. The docs refer to these as group ids as well which is why I think that was initially chosen.

https://www.mongodb.com/docs/atlas/reference/api/events-projects-get-all/#request-path-parameters

Would you prefer the param be renamed to `projectID`?

Reply:
Ok, that's fine. Just wanted to make sure it wasn't a mistake.